Zend/zend_compile.c

```diff
@@ -1190,6 +1192,15 @@ static zend_string *resolve_class_name(zend_string *name, zend_class_entry *scop
 
 zend_string *zend_type_to_string_resolved(zend_type type, zend_class_entry *scope) {
 	zend_string *str = NULL;
+
+	uint32_t type_mask = ZEND_TYPE_FULL_MASK(type);
+
+	if (type_mask == MAY_BE_ANY) {
```

Comment:
Why move this to the top, rather than next to the other checks below?

Reply:
You are right, I could move this a little bit more to the below. Also, I removed `VERIFY_RETURN` opcode generation.